server/stycoon/commands.go

```diff
@@ -0,0 +1,50 @@
+package stycoon
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+)
+
+func ParseCommands(s string) (map[string]Command, error) {
+	var c map[string]Command
+	err := json.Unmarshal([]byte(s), &c)
+	return c, err
+}
+
+func (game *Game) ProcessCommands(commands map[string]Command) map[string]string {
+	res := map[string]string{}
+	for strId, command := range commands {
+		id, err := strconv.Atoi(strId)
+		if err != nil {
+			res[strId] = fmt.Sprintf("unable to convert id to number")
+			continue
+		}
+		log.Info().Msgf("id: %d, command_type: %s", id, command.Type)
```

Comment:
I think we should check that `id` is a ship owned by the player here. Or do we validate this somewhere else?

Reply:
yeah that was my concern too, we should definitely validate it

Reply:
Done - however, not tested